src/examples/monotonicity.csv

```diff
@@ -0,0 +1,792 @@
+"subset_eq(A, A)",1
+"subset_eq(A, difference(A, B))",1.0

```

Comment:
I think something is slightly off here: I don't think this one should be upward monotone, and I think you're only measuring upward monotonicity right now (correct me if this is wrong).  For instance: $A \subseteq A \setminus B$ is true only if $A \cap B = \emptyset$, so that $A \setminus B = A$.  But in that case, it's possible for there to be models with $B \subseteq B'$ and $A \cap B \neq \emptyset$, in which case thsi quantifier would be false, so it shuoldn't be upward monotone

Reply:
This _might_ be onnected to the substructure stuff as first argument to the monotonicity calculation?

Reply:
You are correct, I had not done worked to get anything other than upward monotonicity implemented yet.
Thanks for finding that and spelling that out. I'll see if I can figure out how to rectify this